Hey 👋🏻 @dongwonmoon,  

This isn’t exactly where the tests are failing. These are just warnings that invoke a warning message but don’t raise an error.  

The tests are actually failing because of some other code that was recently added to the codebase. More specifically:  

```
axelrod/strategies/frequency_analyzer.py:65: error: Need type annotation for "frequency_table" (hint: "frequency_table: dict[<type>, <type>] = ...")  [var-annotated]
Found 1 error in 1 file (checked 1 source file)
```

In another strategy that was introduced recently, one of the attributes’ types is not specified, and this is causing the MyPy tests to fail. Could you tweak line 65 in `strategies/frequency_analyzer.py` (see: [link](https://github.com/Axelrod-Python/Axelrod/blob/dev/axelrod/strategies/frequency_analyzer.py)) to:  

```python
self.frequency_table: dict = dict()
```

The tests should pass after that 😅 (I’m about 90% sure). I assume it’ll be okay with the rest of the team if you include this fix in this pull request.

--- a/axelrod/strategies/_strategies.py
+++ b/axelrod/strategies/_strategies.py
@@ -193,6 +193,7 @@ from .memoryone import (  # pylint: disable=unused-import
 from .memorytwo import AON2, MEM2, DelayedAON1
 from .memorytwo import MemoryTwoPlayer  # pylint: disable=unused-import
 
+from .momentum import Momentum
 from .mutual import Desperate, Hopeless, Willing
 from .negation import Negation
 from .oncebitten import FoolMeOnce, ForgetfulFoolMeOnce, OnceBitten
@@ -402,6 +403,7 @@ all_strategies = [
     MEM2,
     MathConstantHunter,
     Michaelos,
+    Momentum,
     NTitsForMTats,
     NaiveProber,
     Negation,

--- a/axelrod/strategies/frequency_analyzer.py
+++ b/axelrod/strategies/frequency_analyzer.py
@@ -62,7 +62,7 @@ class FrequencyAnalyzer(Player):
         """
         super().__init__()
         self.minimum_cooperation_ratio = 0.25
-        self.frequency_table = dict()
+        self.frequency_table: dict = dict()
         self.last_sequence = ""
         self.current_sequence = ""
 

--- a/axelrod/strategies/momentum.py
+++ b/axelrod/strategies/momentum.py
@@ -16,12 +16,6 @@ class Momentum(Player):
     Names:
      - Momentum: Original name by Dong Won Moon
 
-    Notes:
-     - While I am an undergraduate student with limited experience in game theory, I
-       believe this strategy has potential in various scenarios.
-     - I encourage experts to explore and extend this idea in other contexts, such as
-       environments with noise, one-hot vectorization approaches at multiple actions.
-
     """
 
     name = "Momentum"

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -53,7 +53,7 @@ Count the number of available players::
 
     >>> import axelrod as axl
     >>> len(axl.strategies)
-    241
+    242
 
 Create matches between two players::
 

--- a/docs/reference/strategy_index.rst
+++ b/docs/reference/strategy_index.rst
@@ -76,6 +76,8 @@ Here are the docstrings of all the strategies in the library.
    :members:
 .. automodule:: axelrod.strategies.memoryone
    :members:
+.. automodule:: axelrod.strategies.momentum
+   :members:
 .. automodule:: axelrod.strategies.meta
    :members:
 .. automodule:: axelrod.strategies.mutual

--- a/run_mypy.py
+++ b/run_mypy.py
@@ -41,6 +41,7 @@ modules = [
     "axelrod/strategies/mathematicalconstants.py",
     "axelrod/strategies/memoryone.py",
     "axelrod/strategies/memorytwo.py",
+    "axelrod/strategies/momentum.py",
     "axelrod/strategies/mutual.py",
     "axelrod/strategies/negation.py",
     "axelrod/strategies/oncebitten.py",

